phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -76,6 +79,11 @@ public RepositoryDTO fetchInstallableApplications() {
         }
     }
 
+    @Override
+    public boolean isSafe() {
```

Comment:
For example, let's say I want to add a local repository, then I want to decide if the content of this repository should have access to my POL 5 installation or not.